### DISCUSSION TITLE
如何在本機測試草稿時取消加入 GA 追蹤碼？

### DISCUSSION THREAD

**Discussion by wastu01** [CaiJimmy/hugo-theme-stack#888] [Q&A] [answered]
找不到任何官方文件說明

**Comment by CaiJimmy** [comment, marked as answer]
https://github.com/CaiJimmy/hugo-theme-stack/blob/bdb9e7fc007a665fadb28055bee6e387cd68719b/layouts/partials/head/head.html#L25C15-L25C46

Stack 直接使用了 Hugo 自带的 GA 模板：`_internal/google_analytics.html`，具体的实现在这里：https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/google_analytics.html

我看了一下，没有判断是不是在本地访问。所以需要自己复制一份 `_internal/google_analytics.html` 然后加入这一逻辑。主题暂时没有实现此功能。

**Comment by wastu01** [comment]
```
<!-- GA -->

{{ if not .Site.IsServer }}
{{ template "_internal/analytics-gtag.html" . }}
{{ end }}

```
